source/administration/config-settings.rst

```diff
@@ -8,6 +8,9 @@ Mattermost configuration settings are maintained in the configuration file ``con
 
 The default location of ``config.json`` is in the ``mattermost/config`` directory. Mattermost must have write permissions to ``config.json``, otherwise changes made in the System Console will have no effect.
 
+**Configuration in Database**
```

Comment:
This section will be probably need to be added here too: https://docs.mattermost.com/administration/prev-config-settings.html.

Reply:
added - thanks for the suggestion